modules/home/helpers/resources.ts

```diff
@@ -0,0 +1,119 @@
+export enum ResourceCategory {
```

Comment:
instead of in a helpers file i would put it under a component folder, because this info only affects to that component.

components/ResourcesLanding/resources.tsx